info.js

```diff
@@ -0,0 +1,96 @@
+// import data object {"caveat": {}, "veneer": {}}
```

Comment:
why so many info files?))

Reply:
it is the result of playing with webpack. It is clicking without knowledge process)